content/13.conclusion.md

```diff
@@ -1,13 +1,12 @@
+Despite these concerns, deep learning techniques have the potential to add value across a diverse array of research and healthcare contexts if they are leveraged in ethical and responsible ways. After all, these techniques have already shown a remarkable ability to meet or exceed the performance of human effort and older algorithms across numerous fields and subdisciplines.
+Beyond merely achieving good predictive performance in certain tasks, deep learning also has the potential to uncover hidden biological and clinical patterns of potentially high impact, which could fundamentally drive research discoveries and the delivery of new products to market.
+On the whole, deep learning must be approached with genuine thoughtfulness, caution, and responsibility if it is to reach its full potential.
 
-Through the tips and recommendations provided within this manuscript, we hope to encourage a prudent, vigilant community of computational practitioners, experimental biologists, and clinical scientists: colleagues who, before excitedly stitching together lines of code and datasets, first pause to think, dialogue, plan, and discern how their work might have far-reaching consequences with ethical dimensions.
-This holistic approach will help us to advance accountability, beneficence, and quality in science.
-
-Thus, we aim not only to increase the accessibility of deep learning techniques within the life sciences, but also to improve upon the reproducibility and interpretability of high-quality deep learning research in the literature and scientific community—especially given that published findings, models, and datasets will be leveraged to yield innovative tools, services, and products in the marketplace.
-Indeed, we hope that these tips will serve as a powerful engine for promoting meaningful discussions, reflections, team learnings, and best practices to drive collaboration that fosters cutting-edge deep learning innovation, sensibly and responsibly.
+Through the tips and recommendations provided within this manuscript,, we have tried to increase the accessibility of deep learning techniques within the life sciences, as well as to improve the likeihood of deep learning research that is of high quality, interpretability, and reproducibility. This is of critical importance given that published findings, models, and datasets will be leveraged to yield innovative tools, services, and products across societal spheres.
+We also hope to promote the development of a prudent and vigilant community of computational practitioners, experimental biologists, and clinical scientists that pauses to think, discuss, and carefully plan before excitedly stitching together lines of code that improperly apply deep (and tradition) learning techniques.
```

Comment:
Can we make this sentence more positive? I do want to end on an uplifting note.

Reply:
I tried to edit it to do that - what do you think?